iroha-cli/interactive/impl/interactive_common_cli.cpp

```diff
@@ -95,21 +101,23 @@ namespace iroha_cli {
 
     nonstd::optional<std::vector<std::string>> parseParams(
         std::string line, std::string command_name, ParamsMap params_map) {
-      auto params_description = findInHandlerMap(command_name, params_map);
+      auto params_description = findInHandlerMap(command_name, std::move(params_map));
       if (not params_description.has_value()) {
         // Report no params where found for this command
         std::cout << "Command params not found" << std::endl;
         // Stop parsing, something is not implemented
         return nonstd::nullopt;
       }
-      auto words = parser::split(line);
+      auto words = parser::split(std::move(line));
       if (words.size() == 1) {
         // Start interactive mode
         std::vector<std::string> params;
         std::for_each(
             params_description.value().begin(),
             params_description.value().end(),
-            [&params](auto param) { params.push_back(promtString(param)); });
+            [&params](auto param) {
+              params.push_back(promtString(param).value());
```

Comment:
promptString returns nullptr, but here you use .value() which leads to throw of nullopt dereference exception